### PR TITLE
[FR] White labeling

### LINKED ISSUE #2301: [FR] White labeling
**Problem**
A lot of companies do not want it to be known which software they use. Either for not letting their competitor know what they use or for security reasons.

**Suggested solution**
Removing all tags and graphics that identify InvenTree would be a great option.
Also hiding normal debug and version info from users should be an option.

**Describe alternatives you've considered**
This is nice to have.

**Examples of other systems**
Most 'big vendor' solutions come with a very reduced custom login page (that they charge for like they invented HTML). That should be enough.

**Do you want to develop this?**
Yes

### PR DESCRIPTION
**Pull request by matmair**
This PR adds more customization options to Inventree:
- Add a message to the login page
- Add an info banner to the main page
- Exchange the logo
- Remove the InvenTree name from all user-facing places

Fixes #2301

<a href="https://gitpod.io/#https://github.com/inventree/InvenTree/pull/2861"></a>

### CODE REVIEW

**Comment by matmair** [comment]
@SchrodingersGat this will require slight changes to the demo instance: the demo texts were removed from the code-base and moved to the config template. Please just remove the comments on the new lines there and everything should be back as before

File: InvenTree/InvenTree/settings.py [hunk 1]
```diff
@@ -912,3 +913,21 @@ def _is_true(x):
...
 PLUGIN_FILE_CHECKED = False                    # Was the plugin file checked?
+
+# user interface customization values
+CUSTOMIZE = get_setting(
+    'INVENTREE_CUSTOMIZE',
+    CONFIG.get('customize', {}),
+    {}
+)
+
+CUSTOM_LOGO = get_setting(
```
Comment (SchrodingersGat) [inline InvenTree/InvenTree/settings.py:924]: This option will (by default) throw a warning, right? as 'INVENTREE_LOGO' will not exist in the media dir.

Should the default value be an empty string (''), or the line commented out?

**Comment by SchrodingersGat** [comment]
Looks pretty reasonable, I'll pull it down and have a closer look.

File: InvenTree/InvenTree/settings.py (hunk 1 above)
Comment (matmair) [reply InvenTree/InvenTree/settings.py:924]: Not exactly. As custom logo is false by default it is not checked. So everything keeps functioning as it used to be.

Comment (matmair) [reply InvenTree/InvenTree/settings.py:924]: @SchrodingersGat can I mark this as resolved?

**Comment by SchrodingersGat** [comment]
### Navbar Message

Navbar message alignment can be tweaked a bit, should match the current demo instance:

image

image

File: InvenTree/InvenTree/settings.py [hunk 2]
```diff
@@ -912,3 +913,21 @@ def _is_true(x):
...
+    'INVENTREE_LOGO',
+    CONFIG.get('logo', False),
```
Comment (SchrodingersGat) [inline InvenTree/InvenTree/settings.py:926]: I would suggest that the custom logo should be specified in the "customize" section of the config file, making this line

```python
CUSTOM_LOGO = get_setting(
    'INVENTREE_CUSTOM_LOGO',
    CUSTOMIZE.get('logo', False)
)
```

Then, the `config.yaml` file could look like:

```yaml
customize:
  navbar_message: My custom navbar message
  login_message: My custom login message
  logo: media/my_custom_logo.png
```

**Comment by matmair** [comment]
> ### Navbar Message
> 
> Navbar message alignment can be tweaked a bit, should match the current demo instance, centred vertically and horizontally
> 
> image
> 
> image

I have changed the template example to use h6 tags so that the size fits the current method. The result between the demo and this PR are the same now.

**Comment by SchrodingersGat** [comment]
@matmair this looks great now, thanks for the updates. Can you add some documentation when you have the time?

**Comment by matmair** [comment]
> @matmair this looks great now, thanks for the updates. Can you add some documentation when you have the time?

@SchrodingersGat PR is already in

**Pull request merged**